cves/CVE-2016-2162.yml

```diff
@@ -119,19 +127,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: |
+    I looked through every link from the CVE tracker, and also searched for the
+    reporter, the CVE, and for any timeline associated with it. Not much came up
+    that was not already on the initial CVE page (or linked pages).
   date:
   automated:
-  apache:
-  contest:
+  apache: false
+  contest: false
 
 subsystem:
   question: |
     What subsystems was the mistake in?
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
```

Comment:
It appears to be part of the "interceptor" package of "struts-core".  It used to be part of "xwork2", but has since been moved.